src/routes/create-klage.tsx

```diff
@@ -56,12 +57,12 @@ const CreateKlage = () => {
     return <Redirect to={`/${klage.id}/begrunnelse`} />;
 };
 
-function formatError({ tema, ytelse, saksnummer }: NewKlage): string {
+function formatError({ tema, ytelse, internalSaksnummer }: NewKlage): string {
     let error = `Klarte ikke opprette klage med tema "${tema}"`;
-    if (saksnummer === null) {
+    if (internalSaksnummer === null) {
```

Comment:
Trengs det en egen formatError for userSaksnummer?

Reply:
> Trengs det en egen formatError for userSaksnummer?

Usikker på om det trengs fordi `create-klage` bare brukes når man går inn på `/ny?...`-URLen. Da vil vel `userSaksnummer` alltid være `null`?

Reply:
Sant! Var en kontekst her jeg ikke hadde.